inc/admin/metaboxes/namespace.php

```diff
@@ -94,14 +94,22 @@ function upload_cover_image( $pid, $post ) {
 	}
 
 	list( $width, $height ) = getimagesize( $image['file'] );
-	if ( $width < 625 || $height < 625 ) {
-		$_SESSION['pb_notices'][] = sprintf( __( 'Your cover image (%1$s x %1$s) is too small. It should be 625px on the shortest side.', 'pressbooks' ), $width, $height );
+	if ( $height < 800 ) {
+		$_SESSION['pb_errors'][] = sprintf( __( 'Your cover image was not saved because it is too small (%1$s x %2$s). It should be at least 800px in height.', 'pressbooks' ), $width, $height );
+		return; // Do not save uploaded image if it triggers a size-related error
 	}
 
-	$filesize = filesize( $image['file'] );
-	if ( $filesize > 2000000 ) {
-		$filesize_in_mb = \Pressbooks\Utility\format_bytes( $filesize );
-		$_SESSION['pb_notices'][] = sprintf( __( 'Your cover image (%s) is too big. It should be no more than 2MB.', 'pressbooks' ), $filesize_in_mb );
+	$ratio = number_format( $width / $height, 2 );
+	if ( $ratio > 1 || $ratio < .66 ) {
+		$_SESSION['pb_errors'][] = sprintf( __( 'Your cover image was not saved because the aspect ratio (%s) is outside the permitted range (.66 to 1). We recommend an aspect ratio of .75.', 'pressbooks' ), $ratio );
+		return; // Do not save uploaded image if it triggers a size-related error
+	}
+
+	// resize image without cropping or altering aspect ratio
+	$img = wp_get_image_editor( $image['file'] );
```

Comment:
No tests were included for this resizing action. The function did not previously have any unit tests. Open to ideas for how to write better tests here, @fdalcin